app/views/static/references.html.erb

```diff
@@ -9,30 +9,81 @@
       </div>
 
       <div class='static-page__row'>
+        <h2 class='static-page__subtitle static-page__subtitle--sinai'>Catalogues and Handlists</h2>
 
-        <p class='static-page__text static-page__text--sinai'>The following catalogues and handlists were consulted in creating the descriptive metadata for the Syriac and Arabic manuscripts published through the Sinai Manuscripts Digital Library website.</p>
+        <p class="static-page__text static-page__text--sinai">The following catalogues and handlistswere consulted in creating the descriptive metadata for the manuscripts published through the Sinai Manuscripts Digital Library website.</p>
 
-        <p class='static-page__text static-page__text--sinai'>Atiya = Atiya, Aziz Suryal. <i>The Arabic Manuscripts of Mount Sinai: A Hand-List of the Arabic Manuscripts and Scrolls Microfilmed at the Library of the Monastery of St. Catherine, Mount Sinai.</i> Johns Hopkins Press, 1955. [<%= link_to 'Internet Archive', "https://archive.org/details/arabicmanuscript0000atiy_m0l2/mode/2up", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>Aland = Aland, K. <i>Kurzgefasste Liste der griechischen Handschriften des Neuen Testaments</i>. Berlin: De Gruyter, 1963, rev. ed. 1994. [<%= link_to 'WorldCat', 'https://www.worldcat.org/oclc/32035214&amp;sa=D&amp;source=editors&amp;ust=1683922505975654&amp;usg=AOvVaw2RrAa4ir7so9m1wEDFxrFp"', class: 'si-link' %>]</p>
```

Comment:
I noticed this throughout, but the URLs in the links have a long string of parameters add at the end e.g. `&amp;sa=D&amp;source=editors&amp;ust=1683922505975654&amp;usg=AOvVaw2RrAa4ir7so9m1wEDFxrFp"`. In most cases this is causing the links not to resolve

Reply:
I believe all of the links on this page have been affected

Reply:
In this case it should be `https://www.worldcat.org/oclc/32035214`. I can go through and include the correct links for each